metecho/api/jobs.py

```diff
@@ -1156,3 +1168,109 @@ def user_reassign(scratch_org, *, new_user, originating_user_id):
 
 
 user_reassign_job = job(user_reassign)
+
+
+def refresh_datasets(task: Task, originating_user_id=None):
+    """
+    Refresh the dataset definition cache on `task` by parsing definitions from the Task branch
+    """
+    try:
+        task.refresh_from_db()
+        with local_github_checkout(
+            repo_owner=task.root_project.repo_owner,
+            repo_name=task.root_project.repo_name,
+            commit_ish=task.branch_name,
+        ) as project_path:
+            errors = []
+            definitions = {}
+            datasets_dir = Path(project_path) / "datasets"
+            if datasets_dir.exists():
+                datasets = datasets_dir.iterdir()
+            else:
+                datasets = ()
+                errors.append("Could not find 'datasets/' directory in the Task branch")
```

Comment:
Should these user-facing strings be translated?

Reply:
Fixed in d9f6897cf7c6370dad745d9529c9335fe10a46d9